models/sonar/model.sdf

```diff
@@ -28,7 +28,7 @@
         </material>
       </visual>
       <sensor name="sonar" type="sonar">
-        <pose>0 0 0 0 -1.57 0</pose>
```

Comment:
Actually, this wasn't forward facing before: https://github.com/PX4/sitl_gazebo/pull/339/files#diff-d82a6bf200434ac922abd2c2712fe1f6L28

Reply:
No, originally it was forward facing. 100% sure of this.

Reply:
https://github.com/PX4/sitl_gazebo/pull/339/files#diff-1155e8d444b5184de0a4b9e4d0aa4cb6R248 if you set the `</pose>` as described here, the reported orientation is 0 (forward facing) but the sonar is backward facing!

Reply:
Doesn't matter now. The orientation logic required some changes on the models and sensors. Please make sure your change still guarantees that the protobuf Range msg orientation and the distance_sensor msg orientation fields make sense.

Reply:
> https://github.com/PX4/sitl_gazebo/pull/339/files#diff-1155e8d444b5184de0a4b9e4d0aa4cb6R248 if you set the `</pose>` as described here, the reported orientation is 0 (forward facing) but the sonar is backward facing!

If you take a look at the source code at master, that function is not there anymore.